website/blog/2021-07-26-7.15.0.md

```diff
@@ -0,0 +1,132 @@
+
+Additionally, you can now specify [Rhino](https://mozilla.github.io/rhino/) as a compilation target.
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.15.0).
+
+<!-- truncate -->
+
+> Mention [recent blog post](https://babeljs.io/blog/2021/05/10/funding-update) and quick funding update?
+
+If you or your company want to support Babel and the evolution of JavaScript, but aren't sure how, you can donate to us on our [Open Collective](https://opencollective.com/babel) and, better yet, work with us on the implementation of [new ECMAScript proposals](https://github.com/babel/proposals) directly! As a volunteer-driven project, we rely on the community's support to fund our efforts in supporting the wide range of JavaScript users. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss more!
+
+## Highlights
+
+### ECMAScript features enabled by default
+
+In the last two meetings, the [top-level `await`](https://github.com/tc39/proposal-top-level-await) and [ergonomic brand checks for private fields](https://github.com/tc39/proposal-private-fields-in-in) proposals reached Stage 4.
+
+```javascript
+import * as db from "database";
+
+await db.connect(); // top-level await
+
+class DBConnector {
+  #password;
+  static isConnector(obj) {
+    return #password in obj; // ergonomic brand checks
+  }
+}
+```
+
+Babel now supports them by default, so you can remove the following plugins from your configuration:
+- `@babel/plugin-syntax-top-level-await`
+- `@babel/plugin-syntax-private-property-in-object`
+- `@babel/plugin-proposal-private-property-in-object`
+
+Please note that Babel cannot transform top-level `await`, but it can only parse it.
+
+### New TypeScript features ([#13324](https://github.com/babel/babel/pull/13324), [#13528](https://github.com/babel/babel/pull/13528))
+
+[TypeScript 4.4](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/) doesn't include any new syntax that we had to implement, other than a minor restriction: you cannot specify the value of an abstract class field.
+
+```typescript=
+abstract class C {
+    abstract prop = 1; // now a SyntaxError!
+}
+```
+
+However, we did add two TypeScript features that we have been missing for a long time: `const` enums and namespace aliases (`import Alias = Namespace`).
+
+Previously we presented an error when using `const` enums since it requires type information to compile correctly. As a workaround, the community built plugins such as [`babel-plugin-const-enum`](https://github.com/dosentmatter/babel-plugin-const-enum#readme). So Babel now ignores the `const` modifier when compiling enums: this matches TypeScript's behavior when using the `--isolatedModules` options.
```

Comment:
```suggestion
Previously we presented an error when using `const` enums since it requires type information to compile correctly. As a workaround, the community built plugins such as [`babel-plugin-const-enum`](https://github.com/dosentmatter/babel-plugin-const-enum#readme). Babel now ignores the `const` modifier when compiling enums, which matches TypeScript's behavior when using the `--isolatedModules` option.
```